NitroxServer/App.config

```diff
@@ -10,5 +10,6 @@
         <add key="GameMode" value="Survival"/>
         <add key="DisableConsole" value="false"/>
         <add key="ServerAdminPassword" value=""/>
+        <add key="SaveFile" value="save.nitrox"/>
```

Comment:
Maybe make the save name configurable, but enforce the `.nitrox` at the end of the string?

Reply:
Maybe change the setting to something like world name and then combine the world name and the file extension as the save file.

Reply:
Yea thats what I meant

Reply:
Done